src/components/visualizations/chart/test/chartOptionsBuilder.spec.ts

```diff
@@ -3110,26 +3117,26 @@ describe("chartOptionsBuilder", () => {
         });
 
         describe("in usecase of combo chart", () => {
-            it("should assign `line` type to second serie according mbObject", () => {
+            it("should assign `line` type to second series according mbObject", () => {
                 const chartOptions = generateChartOptions(fixtures.comboWithTwoMeasuresAndViewByAttribute, {
-                    type: "combo",
+                    type: VisualizationTypes.COMBO,
```

Comment:
Cache this `VisualizationTypes....` stuff at the top of this file would save some type. 
What do you think?

Reply:
done